substrait/proto/update_proto_includes.pl

```diff
@@ -0,0 +1,6 @@
+#!/bin/perl -w
+
```

Comment:
can we please add a comment about what this does?

Reply:
Done.